chore: upgrade gapic-generator-python to 0.42.2

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/21202808-e3e7-484e-a1fa-d5b954358bde/targets

- [ ] To automatically regenerate this PR, check this box.

PiperOrigin-RevId: 363762006
Source-Link: https://github.com/googleapis/googleapis/commit/8d17d8fafbb87ac64bb3179b99ac34ed41375a51
PiperOrigin-RevId: 362979558
Source-Link: https://github.com/googleapis/googleapis/commit/0dafa3963ef6fcb8a7f5daaa4bec12adb04de518
PiperOrigin-RevId: 362090097
Source-Link: https://github.com/googleapis/googleapis/commit/4b16c60a8fffe213d3a5002f85696fef2b6a8172
PiperOrigin-RevId: 361662015
Source-Link: https://github.com/googleapis/googleapis/commit/28a591963253d52ce3a25a918cafbdd9928de8cf
PiperOrigin-RevId: 361555541
Source-Link: https://github.com/googleapis/googleapis/commit/65d71a60baf9650404a4d9d65f29e9ba8db490d1
PiperOrigin-RevId: 359562873
Source-Link: https://github.com/googleapis/googleapis/commit/07932bb995e7dc91b43620ea8402c6668c7d102c
PiperOrigin-RevId: 359130387
Source-Link: https://github.com/googleapis/googleapis/commit/eabec5a21219401bad79e1cc7d900c1658aee5fd
PiperOrigin-RevId: 357051517
Source-Link: https://github.com/googleapis/googleapis/commit/f3a60f63c13fb434745ea59b990a82d6ffc803b5
PiperOrigin-RevId: 356540804
Source-Link: https://github.com/googleapis/googleapis/commit/4f60776fe99f1fd8261b6a0493a5f5f4d7e8d969
PiperOrigin-RevId: 356380191
Source-Link: https://github.com/googleapis/googleapis/commit/84cf54e45ed5970980ae868e0a1e5ad1266a8830
PiperOrigin-RevId: 356330876
Source-Link: https://github.com/googleapis/googleapis/commit/c1e59709c1d28795fe9b70eb479579556056bfad
PiperOrigin-RevId: 356328938
Source-Link: https://github.com/googleapis/googleapis/commit/78e0057d81c6969507bf1195b5aad8ac3e7feafd
PiperOrigin-RevId: 355923884
Source-Link: https://github.com/googleapis/googleapis/commit/5e3dacee19405529b841b53797df799c2383536c
PiperOrigin-RevId: 355915985
Source-Link: https://github.com/googleapis/googleapis/commit/35ce99fec72979f6f9b2a5adae835a49648a3231
PiperOrigin-RevId: 355059873
Source-Link: https://github.com/googleapis/googleapis/commit/6ef9eaea379fc1cc0355e06a5a20b594543ee693

tswast commented on .kokoro/test-samples.sh:
```diff
@@ -87,11 +87,11 @@ for file in samples/**/requirements.txt; do
 
-    # If this is a periodic build, send the test log to the FlakyBot.
-    # See https://github.com/googleapis/repo-automation-bots/tree/master/packages/flakybot.
+    # If this is a periodic build, send the test log to the Build Cop Bot.
```
Out-of-date. Maybe needs https://github.com/googleapis/python-datacatalog/pull/120 first

yoshi-automation committed:
changes without context

        autosynth cannot find the source of changes triggered by earlier changes in this
        repository, or by version upgrades to tools such as linters.

yoshi-automation committed:
feat: Add Pub/Sub endpoints for Cloud Channel API.

PiperOrigin-RevId: 355059873

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Feb 1 17:13:22 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 6ef9eaea379fc1cc0355e06a5a20b594543ee693
Source-Link: https://github.com/googleapis/googleapis/commit/6ef9eaea379fc1cc0355e06a5a20b594543ee693

yoshi-automation committed:
fix: Fix constraint resource pattern annotation

PiperOrigin-RevId: 355915985

Source-Author: Google APIs <noreply@google.com>
Source-Date: Fri Feb 5 13:27:16 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 35ce99fec72979f6f9b2a5adae835a49648a3231
Source-Link: https://github.com/googleapis/googleapis/commit/35ce99fec72979f6f9b2a5adae835a49648a3231

yoshi-automation committed:
chore: update gapic-generator-python

PiperOrigin-RevId: 355923884

Source-Author: Google APIs <noreply@google.com>
Source-Date: Fri Feb 5 14:04:52 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 5e3dacee19405529b841b53797df799c2383536c
Source-Link: https://github.com/googleapis/googleapis/commit/5e3dacee19405529b841b53797df799c2383536c

yoshi-automation committed:
chore: remove non-existent package option in java_gapic_library rules for cloud APIs

Committer: @miraleung
PiperOrigin-RevId: 356328938

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Feb 8 12:39:42 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 78e0057d81c6969507bf1195b5aad8ac3e7feafd
Source-Link: https://github.com/googleapis/googleapis/commit/78e0057d81c6969507bf1195b5aad8ac3e7feafd

yoshi-automation committed:
feat: Make resolution status field available for error groups. Now callers can set the status of an error group by passing this to UpdateGroup. When not specified, it's treated like OPEN. feat: Make source location available for error groups created from GAE.

PiperOrigin-RevId: 356330876

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Feb 8 12:48:44 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: c1e59709c1d28795fe9b70eb479579556056bfad
Source-Link: https://github.com/googleapis/googleapis/commit/c1e59709c1d28795fe9b70eb479579556056bfad

yoshi-automation committed:
feat: added ApplySoftwareUpdate API docs: various clarifications, new documentation for ApplySoftwareUpdate chore: update proto annotations

PiperOrigin-RevId: 356380191

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Feb 8 16:30:59 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 84cf54e45ed5970980ae868e0a1e5ad1266a8830
Source-Link: https://github.com/googleapis/googleapis/commit/84cf54e45ed5970980ae868e0a1e5ad1266a8830

yoshi-automation committed:
fix: Remove dependency on AppEngine's proto definitions. This also removes the source_references field.

PiperOrigin-RevId: 356540804

Source-Author: Google APIs <noreply@google.com>
Source-Date: Tue Feb 9 10:53:59 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 4f60776fe99f1fd8261b6a0493a5f5f4d7e8d969
Source-Link: https://github.com/googleapis/googleapis/commit/4f60776fe99f1fd8261b6a0493a5f5f4d7e8d969

yoshi-automation committed:
docs: Update rules for currency_code in budget_amount.

PiperOrigin-RevId: 357051517

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu Feb 11 13:54:03 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: f3a60f63c13fb434745ea59b990a82d6ffc803b5
Source-Link: https://github.com/googleapis/googleapis/commit/f3a60f63c13fb434745ea59b990a82d6ffc803b5

yoshi-automation committed:
feat: Publish new fields to support Customer Managed Encryption Keys (CMEK) on the existing Cloud Bigtable service methods.

PiperOrigin-RevId: 359130387

Source-Author: Google APIs <noreply@google.com>
Source-Date: Tue Feb 23 14:08:20 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: eabec5a21219401bad79e1cc7d900c1658aee5fd
Source-Link: https://github.com/googleapis/googleapis/commit/eabec5a21219401bad79e1cc7d900c1658aee5fd

yoshi-automation committed:
chore: update gapic-generator-python to 0.40.11

PiperOrigin-RevId: 359562873

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu Feb 25 10:52:32 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 07932bb995e7dc91b43620ea8402c6668c7d102c
Source-Link: https://github.com/googleapis/googleapis/commit/07932bb995e7dc91b43620ea8402c6668c7d102c

yoshi-automation committed:
feat: add PHP µ-generator build targets to googleads

Committer: @aohren
PiperOrigin-RevId: 361555541

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 8 07:00:53 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 65d71a60baf9650404a4d9d65f29e9ba8db490d1
Source-Link: https://github.com/googleapis/googleapis/commit/65d71a60baf9650404a4d9d65f29e9ba8db490d1

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.42.2

PiperOrigin-RevId: 361662015

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 8 14:47:18 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 28a591963253d52ce3a25a918cafbdd9928de8cf
Source-Link: https://github.com/googleapis/googleapis/commit/28a591963253d52ce3a25a918cafbdd9928de8cf

yoshi-automation committed:
feat: added fallback option when restoring an agent docs: clarified experiment length

PiperOrigin-RevId: 362090097

Source-Author: Google APIs <noreply@google.com>
Source-Date: Wed Mar 10 10:50:48 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 4b16c60a8fffe213d3a5002f85696fef2b6a8172
Source-Link: https://github.com/googleapis/googleapis/commit/4b16c60a8fffe213d3a5002f85696fef2b6a8172

yoshi-automation committed:
feat:added mosaic layout docs:clarified alignment_period max value and updated IAM docs link

PiperOrigin-RevId: 362979558

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 15 10:55:19 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 0dafa3963ef6fcb8a7f5daaa4bec12adb04de518
Source-Link: https://github.com/googleapis/googleapis/commit/0dafa3963ef6fcb8a7f5daaa4bec12adb04de518

yoshi-automation committed:
fix!: remove rpc or fields that are unintended to release fix!: remove StreamingAnalyzeContent, CreateCallMatcher, ListCallMatchers, DeleteCallMatcher rpc from v2/v2beta1 fix!: remove `input_audio` field from AnalyzeContentRequest from v2/v2beta1 fix!: remove proto message CreateCallMatcherRequest, CreateCallMatcherResponse, ListCallMatchersRequest, ListCallMatchersResponse, DeleteCallMatcherRequest, DeleteCallMatcherResponse, CallMatcher, StreamingAnalyzeContentRequest, StreamingAnalyzeContentResponse, AudioInput from v2/v2beta1, TelephonyDtmfEvents, TelephonyDtmf from v2

Committer: @sheimi
PiperOrigin-RevId: 363762006

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu Mar 18 15:37:05 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 8d17d8fafbb87ac64bb3179b99ac34ed41375a51
Source-Link: https://github.com/googleapis/googleapis/commit/8d17d8fafbb87ac64bb3179b99ac34ed41375a51
